### DISCUSSION TITLE
Relational Column display conditions

### DISCUSSION THREAD

**Discussion by tjhunkin-inhance** [rappasoft/laravel-livewire-tables#1647] [Q&A]
How do I show a column where if the nickname is filled in display that, if not, display a concat of name and surname for a related table?

**Comment by lrljoe** [comment]
So if you use a format() callback, you can do that in there.

If you share the current code for your Column, I'll give you some pointers.

I'm more active on the Discord than on here, so feel free to nudge me on there!

**Comment by tjhunkin-inhance** [comment]
probably in conjunction with

```php
$this->setAdditionalSelects(['users.id as id']);
```
